content/php/concepts/string-functions/terms/htmlentities/htmlentities.md

```diff
@@ -0,0 +1,122 @@
+    | ------------- | ----------------------------------- |
+    | ENT_COMPAT    | Default. htmlentities() function encodes only double quotes from `$string`|
+    | ENT_QUOTES    | htmlentities() function encodes double and single quotes from `$string`   |
+    | ENT_NOQUOTES  | htmlentities() function does not encode any quotes from `$string`|
+
+  - Invalid encoding:
+
+    | Constant Name  | Description |
+    |---|---|                                                                                                                                    
+    | ENT_IGNORE| htmlentities() function ignores invalid encoding instead of having the function return an empty string. Should be avoided, as it may have security|                        
+    | ENT_SUBSTITUTE | htmlentities() function replaces invalid encoding for a specified character set with a Unicode Replacement Character U+FFFD (UTF-8) or &#FFFD; instead of returning an empty string.s |
+    | ENT_DISALLOWED | htmlentities() function replaces code points that are invalid in the specified doctype with a Unicode Replacement Character U+FFFD (UTF-8) or &#FFFD;|                
+
+  - Additional flags for specifying the used doctype:
+
+    | Constant Name | Description|
+    | ------------- | --------------------------------- |
+    | ENT_HTML5     | htmlentities() function handles code as HTML 5|
+    | ENT_XHTML     | htmlentities() function handles code as XHTML|
+
+
+  - Character sets that are supported:
+    |Charset|Aliases|Description|
+    |---|---|---|
+    |ISO-8859-1|ISO8859-1|htmlentities() function works with Western European, Latin-1 charset.|
+    |ISO-8859-15|ISO8859-15|htmlentities() function works with Western European, Latin-9. Adds the Euro sign, French and Finnish letters missing in Latin-1 (ISO-8859-1) charset.|
+    |UTF-8||htmlentities() function works with ASCII compatible multi-byte 8-bit Unicode charset.|
+    |cp866|ibm866, 866|htmlentities() function works with DOS-specific Cyrillic charset.|
+    |cp1251|Windows-1251, win-1251, 1251|htmlentities() function works with Windows-specific Cyrillic charset.|
+    |cp1252|Windows-1252, 1252|htmlentities() function works with Windows specific charset for Western European chatset.|
+    |MacRoman||htmlentities() function works with charset that was used by Mac OS.|
+
+    > **Note:** Any other character sets are not recognized. The default encoding will be used instead and a warning will be emitted.
+
+
+> **Note:** The `htmlentities()` function returns the converted `$string`. However, if the `$string` parameter contains invalid encoding, it will return an empty `$string`, unless either the `ENT_IGNORE` or `ENT_SUBSTITUTE` flags are set
+
+**Entities that will be encoded**:
+
+|Characters|Convert to|Result|
+|---|---|---|
+|& (ampersand)|becomes|\&amp;|
+|" (double quote)|becomes|\&quot;|
+|' (single quote)|becomes|\&#039;|
+|< (less than)|becomes|\&lt;|
+|> (greater than)|becomes|\&gt;|
```

Comment:
There's an HTML entities entry with a table of entities. The note is good, just add a period to the end of the note.